packages/number-input/src/NumberInput/NumberInput.tsx

```diff
@@ -143,13 +146,18 @@ export const NumberInput = React.forwardRef<HTMLInputElement, NumberInputProps>(
             )}
           </div>
           <div
-            className={errorMessageWrapperStyles}
+            id={errorMessageId}
+            className={cx(errorContainerStyles, {
+              [errorContainerMarginTop]: showError,
+            })}
```

Comment:
do the below aria-attributes match what we have in FormField? If not let's make them match, bc we want to make the eventual backport to use that component as seamless as possible.

Reply:
made a shareable `FormFieldFeedback` component that will make sure the aria-attributes match across inputs: https://github.com/mongodb/leafygreen-ui/pull/2318/files#diff-226086e5a2e4eeb4bcf2ebfbbfc394693b53ea7cf5eff665b5afcd846637741aR55-R56